### ISSUE TITLE
Can the loss become negative?

### ISSUE THREAD

**Issue by thancaocuong** [orhir/STG-NF#13]
As I training on my custom dataset, the loss become negative with default setting using only normal data (unsupervised approach). So do you think it's a problem?

**Comment by orhir** [comment]
Hi, the loss is the negative log likelihood of the training data, thus it **can** be negative.

**Comment by thancaocuong** [comment]
thank you so much
